docs/site/Decorators_repository.md

```diff
@@ -137,13 +137,128 @@ added here as this feature progresses._
 
 Syntax:
 
-- `@belongsTo`
-- `@hasOne`
-- `@hasMany`
+- `@belongsTo(targetResolver: EntityResolver<T>, definition?: Partial<BelongsToDefinition>)`
+  - Many-to-one connection between models.
+  - A `Todo` belongs to a `TodoList`.
+  - See [here](BelongsTo-relation.md) for more details.
+
+{% include code-caption.html content="todo.model.ts" %}
+
+```
```

Comment:
First of all, please add `ts` to mark the source code as TypeScript.

Secondly, this code snippet is rendered as part of the main text flow, not inside the first item in your list of decorators. To fix that, you need to indent the entire code snippet (including triple-back-ticks before and after the code) by 4 spaces - that's how Markdown works.

However, in this case, let's use a different approach. Because there is quite a lot of content for each decorator, I feel it's much better to create a new subsection for each decorator. Not only the formatting will be easier, but more importantly there will be anchors allowing people to create URLs pointing to docs for a specific relation decorator (e.g. `@belongsTo`).

I am proposing the following layout:

```markdown
### Relation Decorators

#### Relation Decorator

#### BelongsTo Decorator

#### HasMany Decorator

(etc.)
```